integration/resource_lacework_integration_inline_scanner_test.go

```diff
@@ -0,0 +1,42 @@
+package integration
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+// TestIntegrationInlineScannerCreate applies integration terraform:
+// => '../examples/resource_lacework_integration_inline_scanner'
+//
+// It uses the go-sdk to verify the created integration,
+// applies an update with new integration name and destroys it
+func TestIntegrationInlineScannerCreate(t *testing.T) {
+	tokenName := fmt.Sprintf("Inline Scanner Token Terraform - %s", time.Now())
+
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: "../examples/resource_lacework_integration_inlineScanner",
+		EnvVars:      tokenEnvVar,
+		Vars: map[string]interface{}{
+			"name": tokenName,
+		},
+	})
+	defer terraform.Destroy(t, terraformOptions)
+
+	// Create new Github Container Registry
+	create := terraform.InitAndApplyAndIdempotent(t, terraformOptions)
+	createData := GetContainerRegisteryInlineScanner(create)
+	assert.Equal(t, tokenName, createData.Data.Name)
+	// assert.Equal(t, []map[string]string{{"foo": "bar"}}, createData.Data.Data.IdentifierTag)
```

Comment:
Would be great to have this assertion included.

Reply:
added back these assertion tests